packages/sdk/src/BoostCore.test.ts

```diff
@@ -1279,3 +1279,85 @@ describe("ERC20PeggedVariableCriteriaIncentive Top-Ups", () => {
     expect(newLimit).toBe(originalLimit + expectedNetTopup);
   });
 });
+
+describe("ERC20PeggedVariableCriteriaIncentive with LimitedSignerValidator", () => {
+  test("enforces validator claim limit", async () => {
+    // biome-ignore lint/style/noNonNullAssertion: we know this is defined
+    const referrer = accounts.at(1)!.account!;
+    const signer = accounts.at(0)!;
```

Comment:
if you want to avoid this lint comment, just use an index access
```suggestion
    // biome-ignore lint/style/noNonNullAssertion: we know this is defined
    const referrer = accounts[1].account!;
    const signer = accounts[0]!;
```